services/base-xml.js

```diff
@@ -26,7 +32,7 @@ class BaseXmlService extends BaseService {
         underlyingError: validateResult.err,
       })
     }
-    const xml = fastXmlParser.parse(buffer)
+    const xml = fastXmlParser.parse(buffer, parserOptions)
```

Comment:
As _base-xml.spec.js_ uses real XML fixtures, it should be fairly easy to assert that the parser options are correctly provided to _fastXmlParser_ by writing a test that would fail with its default options. 😉

Reply:
Good idea :-) I've added a test.